src/View/Components/Input.php

```diff
@@ -105,7 +105,7 @@ protected function getDefaultColorClasses(): string
 
     protected function getDefaultClasses(): string
     {
-        return Str::of('block w-full sm:text-sm rounded-md transition ease-in-out duration-100 focus:outline-none')
+        return Str::of('block form-input w-full sm:text-sm rounded-md transition ease-in-out duration-100 focus:outline-none')
```

Comment:
I think that this class can be applied to the input blade layout because the Textarea extends the input. (Heritage is a bad practice, 😞 zero for me!!!)
You can use the `->class` function to merge the default class and the `form-class`
Apply the same into the textarea
https://laravel.com/docs/8.x/blade#conditionally-merge-classes

Ex:
```
<input {{ $attributes->class([
      $getInputClasses($hasError),
      'form-input'
  ])->merge([
      'class'        => $getInputClasses($hasError),
      'type'         => 'text',
      'autocomplete' => 'off',
  ]) }} />
```

Or use in the input like the textarea
```
 <textarea {{ $attributes->merge([
            'class'        => 'form-textarea '. $getInputClasses($hasError),
```